### PR TITLE
docs: update readme

### PR DESCRIPTION
**Pull request by killme2008**
I hereby agree to the terms of the [GreptimeDB CLA](https://gist.github.com/xtang/6378857777706e568c1949c7578592cc)

## What's changed and what's your intention?

Simplify readme.

## Checklist

- [ ]  I have written the necessary rustdoc comments.
- [ ]  I have added the necessary unit tests and integration tests.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -100,64 +100,22 @@ Or if you built from docker:
...
+To write and query data, you can find many [clients](https://docs.greptime.com/user-guide/clients).
 
 ## Resources
 
 ### Installation
 
-- [Pre-built Binaries](https://github.com/GreptimeTeam/greptimedb/releases):
+- [Pre-built Binaries](https://greptime.com/download):
   For Linux and macOS, you can easily download pre-built binaries that are ready to use. In most cases, downloading the version without PyO3 is sufficient. However, if you plan to run scripts in CPython (and use Python packages like NumPy and Pandas), you will need to download the version with PyO3 and install a Python with the same version as the Python in the PyO3 version. We recommend using virtualenv for the installation process to manage multiple Python versions.
+- [Nightly-built Binaries](https://github.com/GreptimeTeam/greptimedb/releases)
```
Comment (sunng87) [inline README.md:117]: nightly downloads are also available on download page

@alili we can add anchors to different sections of donwload page

**Review by xtang — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -100,64 +100,22 @@ Or if you built from docker:
...
-    +-------+--------------------------+------+--------+
-    3 rows in set (0.03 sec)
-    ```
-
-You can always cleanup test database by removing `/tmp/greptimedb`.
+Please see [the online document site](https://docs.greptime.com/getting-started/overview#install-greptimedb) for more installation options and [operations info](https://docs.greptime.com/user-guide/operations/overview).
+
+### Get started
+
+Please see the [complete getting started guide](https://docs.greptime.com/getting-started/overview#connect) on our [official document site](https://docs.greptime.com/).
```
Comment (sunng87) [inline README.md:107]: These two lines are both started with `Please see`

File: README.md [hunk 3]
```diff
@@ -100,64 +100,22 @@ Or if you built from docker:
...
+
+To write and query data, you can find many [clients](https://docs.greptime.com/user-guide/clients).
```
Comment (sunng87) [inline README.md:109]:
```suggestion
To write and query data, GreptimeDB is compatible with multiple [protocols and clients](https://docs.greptime.com/user-guide/clients).
```

**Comment by killme2008** [comment]
@sunng87 PTAL

**Review by sunng87 — APPROVED** [review]

**Pull request merged**